### DISCUSSION TITLE
Welcome to BeatNet Discussions!

### DISCUSSION THREAD

**Discussion by mjhydri** [mjhydri/BeatNet#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fastline-coachyy** [comment]
I am looking for a programmer who can integrate phrase recognition into my new project. Specifically, it involves recognising 32-beat phrases with additional recognition of breaks that are shorter than 32 beats.
Who can help or knows someone who can help?
